test/parallel/test-internal-errors.js

```diff
@@ -118,3 +124,9 @@ assert.throws(() => {
                            type: TypeError,
                            message: /^Error for testing 2/ }));
 }, /AssertionError: .+ does not match \S/);
+
+assert.doesNotThrow(() => errors.E('TEST_ERROR_USED_SYMBOL'));
+assert.throws(
+  () => errors.E('TEST_ERROR_USED_SYMBOL'),
+  /^AssertionError: Error symbol: TEST_ERROR_USED_SYMBOL was used.$/
```

Comment:
The period should be escaped to avoid matching *any* character instead of a literal period.